rpc/v0/params.go

```diff
@@ -90,6 +90,13 @@ type (
 		GasLimit int64  `json:"gas_limit"`
 	}
 
+	// Used when sending a 'Send' transaction.
+	SendParam struct {
+		PrivKey   []byte `json:"priv_key"`
```

Comment:
ah never mind, transact and transactAndHold have the same "broken" architecture; both of these need to be deprectaed in favor of using eris-client which now has the `waitForConfirmation`; but we can merge it for rc3